src/locales/fi.json

```diff
@@ -4,5 +4,11 @@
   },
   "featureX": {
     "hello": "Morjesta featuresta x"
+  },
+  "header": {
+    "home": "Alkuun",
```

Comment:
Sv translations is missing. You can type them ie. "SV: Alkuun" and someone will translate them later.